contrib/format-excel/src/main/java/org/apache/drill/exec/store/excel/ExcelBatchReader.java

```diff
@@ -273,7 +273,12 @@ private int getColumnCount() {
     int columnCount;
 
     if (readerConfig.headerRow >= 0) {
-      columnCount = sheet.getRow(sheet.getFirstRowNum()).getPhysicalNumberOfCells();
+      try {
+        columnCount = sheet.getRow(sheet.getFirstRowNum()).getPhysicalNumberOfCells();
+      } catch (NullPointerException e) {
```

Comment:
It is a bad practice to catch NPE instead of adding a check for null. If I understand the API correctly, `sheet.getRow(sheet.getFirstRowNum())` may return null for the case when row is not defined. Could you please add check for null there?

Reply:
Fixed.  If the `sheet.getFirstRowNum())` is `-1` then the sheet is empty.  I included a check for this.

Reply:
Thanks for fixing it, but I'm afraid we can obtain NPE here for other cases, according to its JavaDoc, null may be returned for the case when a row with specified index wasn't defined. Also, there is a line of code below which may have a possible issue. I would recommend refactoring it to something like this:
```
  private int getColumnCount() {
    int rowNumber = readerConfig.headerRow > 0 ? sheet.getFirstRowNum() : 0;
    XSSFRow sheetRow = sheet.getRow(rowNumber);

    return sheetRow != null ? sheetRow.getPhysicalNumberOfCells() : 0;
  }
```

Reply:
@vvysotskyi 
Thanks for the review.  I replaced the method with your code and all unit tests pass.